extension/src/editor/containers/dataset/blocks/StatusBlock.tsx

```diff
@@ -10,6 +12,8 @@ import {
   EditorTextField,
 } from "../../ui-components";
 
+dayjs.extend(utc);
```

Comment:
Personally this should be invoked in one module if possible.

Reply:
Updated. Thank you!